guides/v2.1/coding-standards/docblock-standard-javascript.md

```diff
@@ -183,7 +185,7 @@ Square.prototype.draw&nbsp;=&nbsp;function()&nbsp;{
 };
 </pre>
 
-<h4 id="frontend-dev-guide-javascript-js-coding-docblock-lends">@lends</h4>
+#### @lends
```

Comment:
Please append `{#frontend-dev-guide-javascript-js-coding-docblock-lends}` to preserve the custom id.